scripts/r2s.py

```diff
@@ -119,14 +119,24 @@ def step1():
                                      flux_tag + '_total',
                                      flux_tag + '_err_total')},
                         meshes_have_mats=False)
+        m = meshtal.tally[tally_num]
+
     geom = config.get('step1', 'geom')
     reverse = config.getboolean('step1', 'reverse')
     num_rays = config.getint('step1', 'num_rays')
     grid = config.getboolean('step1', 'grid')
 
     load(geom)
+
```

Comment:
Probably add a one line descriptive comment here about the following if statement.